From 5d01df56c48d0ef9f08840598a57fa693ac43dcf Mon Sep 17 00:00:00 2001
Message-Id: <5d01df56c48d0ef9f08840598a57fa693ac43dcf.1387298827.git.minovotn@redhat.com>
In-Reply-To: <3ed0fb61a3dc912ef036d7ef450bed192090709e.1387298827.git.minovotn@redhat.com>
References: <3ed0fb61a3dc912ef036d7ef450bed192090709e.1387298827.git.minovotn@redhat.com>
From: "Michael S. Tsirkin" <mst@redhat.com>
Date: Tue, 17 Dec 2013 15:17:47 +0100
Subject: [PATCH 23/56] cleanup object.h: include error.h directly

RH-Author: Michael S. Tsirkin <mst@redhat.com>
Message-id: <1387293161-4085-24-git-send-email-mst@redhat.com>
Patchwork-id: 56329
O-Subject: [PATCH qemu-kvm RHEL7.0 v2 23/57] cleanup object.h: include error.h directly
Bugzilla: 1034876
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>
RH-Acked-by: Marcel Apfelbaum <marcel.a@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: Igor Mammedov <imammedo@redhat.com>

qapi/error.h is simple enough to be included in qom/object.h
direcly and prepares qom/object.h to use Error typedef.

Signed-off-by: Igor Mammedov <imammedo@redhat.com>
Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
(cherry picked from commit 008e05662ae5ff314b2b8462508ffd4b40503369)
---
 include/qom/object.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 include/qom/object.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/qom/object.h b/include/qom/object.h
index 23fc048..c37f3ca 100644
--- a/include/qom/object.h
+++ b/include/qom/object.h
@@ -18,9 +18,9 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include "qemu/queue.h"
+#include "qapi/error.h"
 
 struct Visitor;
-struct Error;
 
 struct TypeImpl;
 typedef struct TypeImpl *Type;
-- 
1.7.11.7

